packages/cli-repl/test/e2e-tls.spec.ts

```diff
@@ -114,9 +112,7 @@ describe('e2e TLS', () => {
         shell.assertContainsOutput('MongoServerSelectionError');
       });
 
-      // TODO: enable when NODE-2977 is solved
-      // eslint-disable-next-line mocha/no-skipped-tests
-      it.skip('fails when not using --tls (connection string)', async() => {
+      it('fails when not using --tls (connection string)', async() => {
```

Comment:
Not sure if actually fixed, it worked when I added a `:` into the filename on Linux but we’ll see what CI says.

Reply:
CI says it’s fixed :)